isisd/isis_cli.c

```diff
@@ -2946,6 +2964,12 @@ DEFPY(isis_ti_lfa, isis_ti_lfa_cmd,
 				"./frr-isisd:isis/fast-reroute/level-1/ti-lfa/node-protection",
 				NB_OP_MODIFY,
 				node_protection ? "true" : "false");
+			nb_cli_enqueue_change(
+				vty,
+				"./frr-isisd:isis/fast-reroute/level-1/ti-lfa/link-fallback",
+				NB_OP_MODIFY,
+				node_protection && link_fallback ? "true"
```

Comment:
nit: the `node_protection` check is unnecessary

Reply:
I agree. Removed!